### ISSUE TITLE
Avoid easily-confused characters in generated user-facing credentials

### ISSUE THREAD

**Issue by pharten** [cloud-gov/cg-site#1184]
When you send passwords for logins, do not use lowercase l,1, or uppercase O,0 at all.

**Comment by pburkholder** [comment]
Hi Paul,

Sorry to hear that this caused issues for you. Which password set/reset process caused you a problem? We have several.

Thanks,

Peter

**Comment by pharten** [comment]
Peter,

From the quickstart page (https://cloud.gov/quickstart/), I was running “cf login -a https://api.fr.cloud.gov -sso” and was prompted for a password.  I went to https://login.fr.cloud.gov/passcode and was prompted to put in my smart card and enter my ID.  When I did this, I was given a password that contained a straight line between two capital letters.  My first interpretation was that it was a lowercase L “l”.  That was not excepted.  My second interpretation was an uppercase i “I”.  That did work.

In some cases, a cut-and-paste of passwords is allowed, and in other cases, it is not.

Thanks,

Paul


Paul Harten, PhD
Computational Environmental Scientist
Emerging Chemistry and Engineering Branch
Land and Materials Management Division
National Risk Management Research Laboratory
US EPA Office of Research and Development
26 West Martin Luther King Drive, MS 443
Cincinnati, Ohio 45268
513-569-7045

http://www2.epa.gov/chemical-research/program-assisting-replacement-industrial-solvents-paris-iii
